src/renderer/vt/XtermEngine.cpp

```diff
@@ -318,10 +347,7 @@ XtermEngine::XtermEngine(_In_ wil::unique_hfile hPipe,
             _lastText = coord;
         }
     }
-    if (_lastText.Y != _lastViewport.ToOrigin().BottomInclusive())

```

Comment:
does this check go away because we're checking `_newBottomLine && position is in bottom line` now?

Reply:
Nah I think it was just a tad bit to defensive on the original implementation of the `_newBottomLine` optimization. Now that we're doing

```c++
    if (coord.Y == _lastViewport.BottomInclusive())
    {
        _newBottomLine = false;
    }
```

in paint.cpp, this old check would clear out the `_newBottomLine` state _too_ often. This one (combined with the paint.cpp change) combine to fix #5039